docs/src/content/docs/tr/reference/overrides.md

```diff
@@ -0,0 +1,375 @@
+
+Mevcut sayfanın içeriğinden önce bir içerik tablosu görüntülemek için kullanılan bir bileşen. 
+Varsayılan uygulama, [`<TableOfContents />`](#tableofcontents) ve [`<MobileTableOfContents />`](#mobiletableofcontents)’i görüntüler.
+
+#### `TableOfContents`
+
+**Varsayılan component:** [`TableOfContents.astro`](https://github.com/withastro/starlight/blob/main/packages/starlight/components/TableOfContents.astro)
+
+Mevcut sayfanın içeriğinin geniş ekranlarda tablosunu görüntüleyen bir bileşen.
+
+#### `MobileTableOfContents`
+
+**Varsayılan component:** [`MobileTableOfContents.astro`](https://github.com/withastro/starlight/blob/main/packages/starlight/components/MobileTableOfContents.astro)
+
+Mevcut sayfanın içeriğinin küçük (mobil) ekranlarda tablosunu görüntüleyen bir bileşen.
+
+---
+
+### İçerik
+
+Bu bileşenler sayfa içeriğinin ana sütununda görüntülenir.
+
+#### `Banner`
+
+**Varsayılan component:** [`Banner.astro`](https://github.com/withastro/starlight/blob/main/packages/starlight/components/Banner.astro)
+
+Her sayfanın üstünde görüntülenen [`banner`](/tr/reference/frontmatter#banner) bileşeni. 
+Varsayılan uygulama, sayfanın banner ön bilgisinin değerine bağlı olarak görüntülenip görüntülenmeyeceğini belirler.
+
+#### `ContentPanel`
+
+**Varsayılan component:** [`ContentPanel.astro`](https://github.com/withastro/starlight/blob/main/packages/starlight/components/ContentPanel.astro)
+
+Ana içerik sütununun bölümlerini sarmak için kullanılan düzen bileşeni.
+
+#### `PageTitle`
+
+**Varsayılan component:** [`PageTitle.astro`](https://github.com/withastro/starlight/blob/main/packages/starlight/components/PageTitle.astro)
+
+Mevcut sayfanın `<h1>` öğesini içeren bir bileşen.
+
+Uygulamalar, varsayılan uygulama gibi `<h1>` öğesine `id="_top"` ayarladıklarından emin olmalıdır.
+
+#### `FallbackContentNotice`
+
+**Varsayılan component:** [`FallbackContentNotice.astro`](https://github.com/withastro/starlight/blob/main/packages/starlight/components/FallbackContentNotice.astro)
+
+Mevcut dil için bir çeviri mevcut olmayan sayfalarda kullanıcıların gördüğü bildirim. 
+Yalnızca çok dilli sitelerde kullanılır.
+
+#### `Hero`
+
+**Varsayılan component:** [`Hero.astro`](https://github.com/withastro/starlight/blob/main/packages/starlight/components/Hero.astro)
+
+Frontmatter’da [`hero`](/tr/reference/frontmatter#hero) ayarlandığında sayfanın üstünde görüntülenen bir bileşen. 
+Varsayılan uygulama büyük bir başlık, slogan ve isteğe bağlı bir resim ile yanı sıra çağrıya yönlendiren bağlantıları gösterir.
+
+#### `MarkdownContent`
+
+**Varsayılan component:** [`MarkdownContent.astro`](https://github.com/withastro/starlight/blob/main/packages/starlight/components/MarkdownContent.astro)
+
+Her sayfanın ana içeriği etrafında görüntülenen bir bileşen. 
+Varsayılan uygulama, Markdown içeriğine uygulanacak temel stilleri ayarlar.
+
```

Comment:
I think this is missing the changes from https://github.com/withastro/starlight/pull/1049/files#diff-ad80506c0d18072cbd9a85e2090c2be41e5a5523fe9f2991a1c653e41d089591

Would you mind adding the missing sentence?

Reply:
i added now